basic-algorithms.js

```diff
@@ -1,7 +1,49 @@
 // Names and Input
+var hacker1 = "Table";
+console.log(hacker1);
+
+var hacker2 = prompt("What's your name?);
+console.log('Navigator´s name is ' + hacker2);
 
 
 //Conditionals
+var driver = "ThomasJames";
+var navigator = "Sally";
 
+if (driver.length > navigator.length){
```

Comment:
Try to be consistent with the format, here a space between the "  ) " and the " { " is missing.